### ISSUE TITLE
I wonder if my vgg npy file broken

### ISSUE THREAD

**Issue by ruiann** [machrisaa/tensorflow-vgg#30]
The range of numbers in the output tensor generated by conv5_4 is very big, can up to 1e5, is this the correct output?

**Comment by ruiann** [comment]
or for different result the numbers have different range? when I try to compare two feature maps by Euclid distance, the loss is very big and able to hide other parts of loss function

**Comment by machrisaa** [comment]
Hi @ruiann, it could be big. It's difficult to tell if your npy file is broken or not. The only thing I will do is to run the [test](https://github.com/machrisaa/tensorflow-vgg/blob/master/test_vgg19.py) to check if it can classify the sample images (tiger and puzzle).

I added this line and check the magnitude of the output of conv5_4:
```
https://github.com/machrisaa/tensorflow-vgg/blob/master/test_vgg19.py
```
The max is just 300-ish for the tiger image. See if you can get the same result or you may need to download the npy again.

**Comment by ruiann** [comment]
@machrisaa yeah the result is ok, but its hard to reduce the Euclid loss of the feature map, I think I need to do more investment, thank you XD
